src/button/default.scss

```diff
@@ -49,25 +49,29 @@
     &:enabled,
     &:enabled:hover,
     &:enabled:active {
-      @include themed {
-        background: linear-gradient(
-          rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-6)),
-          rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-6))
-        ),
-          linear-gradient(t(iui-color-background-1), t(iui-color-background-1));
-      }
+      @include iui-button-active;
+    }
+  }
+}
 
-      > .iui-label {
-        @include themed {
-          color: t(iui-color-foreground-primary);
-        }
-      }
+@mixin iui-button-active {
```

Comment:
I thought we will reuse this mixin inside header button but as I understand it really can't be reused there. So maybe undo this change.